lldb/include/lldb/Target/StackID.h

```diff
@@ -57,15 +57,17 @@ class StackID {
     return *this;
   }
 
-  bool IsCFAOnStack() const { return m_cfa_on_stack; }
+  /// Check if the CFA is on the stack, or elsewhere in the process, such as on
+  /// the heap.
```

Comment:
Should we comment that the caller is now expected to check for Heap frames?

Reply:
Was this meant to be on the `IsYounger` function? If so, I moved the heap check into `IsYounger` to avoid having to put that requirement onto callers.